plugins/lsp/language-servers/rust-analyzer-config.nix

```diff
@@ -37,6 +37,10 @@ let
         lib.types.int
     else if kind == "object" then
       lib.types.attrsOf lib.types.anything
+    else if kind == "submodule" then
+      lib.types.submodule {
+        options = lib.mapAttrs (_: ty: lib.mkOption { type = mkRustAnalyzerType ty; }) typeInfo.options;
```

Comment:
I believe @GaetanLepage spent time ensuring all options have a description to avoid some warning being printed somewhere.

IDK if this matters for submodule sub-options, but perhaps we should have something?

Reply:
I have no idea what description I could provide, an empty description?

Reply:
I think empty description is fine, best to check with @GaetanLepage whether we actually need to do this or not